crates/miden-objects/src/block/mod.rs

```diff
@@ -17,7 +17,12 @@ mod partial_account_tree;
 pub use partial_account_tree::PartialAccountTree;
 
 pub(super) mod account_tree;
-pub use account_tree::{AccountMutationSet, AccountTree};
+pub use account_tree::{
+    AccountMutationSet,
+    AccountTree,
+    account_id_to_smt_key,
+    smt_key_to_account_id,
+};
```

Comment:
nit: I wonder if we should change this to just:
```rust
pub mod account_tree;
```
To keep exports a bit more organized.